api/v1/mongodbcommunity_types.go

```diff
@@ -492,12 +495,37 @@ func (m MongoDBCommunity) GetScramUsers() []scram.User {
 	return users
 }
 
+func (m MongoDBCommunity) IsStillScaling() bool {
+	return scale.IsStillScaling(m) || scale.IsStillScaling(automationConfigReplicasScaler{
+		current: m.CurrentArbiters(),
+		desired: m.DesiredArbiters(),
+	})
+}
+
+// AutomationConfigMembersThisReconciliation determines the correct number of
+// automation config replica set members based on our desired number, and our
+// current number.
 func (m MongoDBCommunity) AutomationConfigMembersThisReconciliation() int {
-	// determine the correct number of automation config replica set members
-	// based on our desired number, and our current number
 	return scale.ReplicasThisReconciliation(automationConfigReplicasScaler{
-		desired: m.Spec.Members,
 		current: m.Status.CurrentMongoDBMembers,
+		desired: m.Spec.Members,
+	})
+}
+
+// AutomationConfigArbitersThisReconciliation determines the correct number of
+// automation config replica set arbiters based on our desired number, and our
+// current number.
+//
+// Will not update arbiters until members have reached desired number.
```

Comment:
> Will not update arbiters until members have reached desired number.

Is this a requirement for the agent to function properly?

Reply:
Yes, better described here: https://github.com/10gen/ops-manager-kubernetes/blob/master/docs/rfcs/0000-supporting-arbiters.md#scaling-of-arbiters

Reply:
sounds good.